src/components/ProjectSettingsTabs/PatchAliasesTab/getFormSchema.ts

```diff
@@ -44,5 +161,70 @@ export const getFormSchema = (
         aliases: patchAliasArray.repoData.uiSchema,
       },
     },
+    patchTriggerAliases: {
+      aliasesOverride: {
+        "ui:widget":
+          projectType === ProjectType.AttachedProject
+            ? widgets.RadioBoxWidget
+            : "hidden",
+        "ui:showLabel": false,
+        "ui:data-cy": "patch-trigger-aliases-override-radio-box",
+      },
+      aliases: aliasesUiSchema,
+      repoData: {
+        aliases: {
+          ...aliasesUiSchema,
+          "ui:readonly": true,
+        },
+      },
+    },
   },
 });
+
+const aliasesUiSchema = {
+  "ui:addButtonText": "Add Patch Trigger Alias",
```

Comment:
Can we make this button add the card to the front of the list instead of the back of the list? If there are already aliases on the page, I can't see the card being added, and it looks like the button doesn't work.
![aliasbutton](https://user-images.githubusercontent.com/13104717/159064495-244ec29d-bcae-4fdc-8245-aa9c5b3f572e.gif)

Reply:
Yeah, I definitely think this pattern makes more sense for the majority of our arrays. It's not immediately obvious how to do this with RJSF so I've opened EVG-16594 to investigate.